lib/std/Build/Step/Run.zig

```diff
@@ -19,10 +20,10 @@ step: Step,
 /// See also addArg and addArgs to modifying this directly
 argv: ArrayList(Arg),
 
-/// Set this to modify the current working directory
+/// Use `setCwd` to modify the current working directory
```

Comment:
Maybe `s/to modify the current working directory/to change the current working directory on process start` to clarify that this is a process global property potentially affecting subprocesses and initially applied etc?